### ISSUE TITLE
Build image for Jetson system

### ISSUE THREAD

**Issue by long-senpai** [JonasVautherin/px4-gazebo-headless#42]
Hello, thanks you for the great work. i used your repo to rebuild px4-gazebo in jetson agx devices. The docker are successful build and running correctly. 
I used mavproxy to forward mavlink packet to other pc.
"python3 /usr/local/bin/mavproxy.py --daemon --master=udpin:127.0.0.1:14550 --out=udp:192.168.1.2:14550  --out=tcpin:0.0.0.0:5760"

However, when i play the video stream in VLC, the stream video is so slow, i mean the frame are freeze and not updated. 
If i ran native in Jetson, everything is working well. 

After some reasearch, i think maybe Gazebo server cannot use GPU in jetson devices if use Display difference 0. 

Do you have any advices? Thank you so much

**Comment by JonasVautherin** [comment]
Yeah the docker container doesn't use the GPU. I never tried to get a docker container to use a GPU, though I've seen online that it was possible.
